makefile

```diff
@@ -0,0 +1,18 @@
+test:
```

Comment:
Nit: I think these files are usually named `Makefile` by default

Reply:
I renamed it but look like GitHub keep it with the lowercase.